sbws/lib/destination.py

```diff
@@ -76,21 +75,26 @@ def connect_to_destination_over_circuit(dest, circ_id, session, cont, max_dl):
             head = requests_utils.head(session, dest.url, verify=dest.verify)
         except (requests.exceptions.ConnectionError,
                 requests.exceptions.ReadTimeout) as e:
+            dest.set_failure
```

Comment:
Using a property to change state makes this code hard to read.
Should it be a function call instead?
This line is repeated 4 times in this function: please fix all of them.

Reply:
Yes, fixed